dcpy/library/templates/lpc_historic_district_areas.yml

```diff
@@ -15,7 +15,7 @@ dataset:
 
   destination:
     geometry:
-      SRS: EPSG:4326
+      SRS: EPSG:2263
```

Comment:
@damonmcc Apologies for leaving this hanging. I don't believe I reran data library after making this change, so I think that next time we run data library with this dataset, this will break what we've got below in the dbt model. 

I this case, I'll just revert this change to keep `EPSG:4326`. cc'ing @fvankrieken and @sf-dcp because it's perhaps a consideration for `extract`/`ingest` for downstream breaking changes like this, and how to deal with them. Supposing we did change the projection, what's the best way to deal with that? Maybe just creating `lpc_historic_district_areas_v2.yml`? Sort of a poor man's versioning scheme 🤷‍♂️

Reply:
I guess it'd better to not revert this change and just change the dbt model when it breaks, since we'd rather archive to `2263` and not have to reproject during the build

Reply:
Looks like it's also used in EDDE, so it's break things there. And even though it's probably trivial to fix there, I think generally best to have consistent projections in data we've archived. Super trivial in this case since we've only ever archived this dataset twice... but also a good opportunity to do things the "right" way with breaking changes